bridges/snowbridge/pallets/outbound-queue-v2/src/lib.rs

```diff
@@ -406,4 +411,18 @@ pub mod pallet {
 			)
 		}
 	}
+
+	impl<T: Config> AddTip for Pallet<T> {
+		fn add_tip(nonce: u64, amount: u128) -> Result<(), AddTipError> {
+			PendingOrders::<T>::try_mutate_exists(nonce, |maybe_order| -> Result<(), AddTipError> {
+				match maybe_order {
+					Some(order) => {
+						order.fee = order.fee.saturating_add(amount);
+						Ok(())
+					},
+					None => Err(AddTipError::NonceConsumed),
```

Comment:
In this case, no such message exists:

```suggestion
					None => Err(AddTipError::UnknownMessage),
```

Reply:
https://github.com/vgeddes/polkadot-sdk/commit/a0fda22f8e4a362f46b101909969750fd2cd6ac7